sync/sync-store/src/main/java/com/duckduckgo/sync/store/dao/SyncAttemptDao.kt

```diff
@@ -29,13 +29,13 @@ interface SyncAttemptDao {
     fun insert(attempt: SyncAttempt)
 
     @Query("SELECT * FROM sync_attempts ORDER BY id DESC LIMIT 1")
-    fun lastAttempt(): SyncAttempt?
+    fun lastAttemptSync(): SyncAttempt?
 
     @Query("DELETE from sync_attempts")
     fun clear()
 
-    @Query("SELECT * FROM sync_attempts ORDER BY id DESC")
-    fun attempts(): Flow<SyncAttempt?>
+    @Query("SELECT * FROM sync_attempts ORDER BY id DESC LIMIT 1")
+    fun lastAttempt(): Flow<SyncAttempt?>
```

Comment:
it was returning a list with all attempts, and the Repository was filtering the last one. We don’t really need that since we only care about the last one.